### DISCUSSION TITLE
Is there a way to disable tight lists in a document?

### DISCUSSION THREAD

**Discussion by fuine** [typst/typst#3289] [Q&A] [answered]
I'm trying to create a presentation using `polylux` and I'd like to customize all the lists to my liking. Specifically, I'd prefer to write a "regular" list (that is without the need for empty lines) and still get it rendered as a non-tight list. I tried globally setting list rendering with `#set list(tight: false, spacing: 1.2em)`, but it doesn't seem to change the way the lists are rendered (without empty lines they are still tight). Is there a way to write:
```typ
- First item
- Second item
- Third item
```
and get it to render non-tight list?

**Comment by laurmaedje** [comment, marked as answer]
The syntax forces `tight: false` which takes precedence over the set rule. There is no way to directly change the value of `tight` at the moment. I'm afraid you'd need to write a list show rule from scratch (the normal lists just desugar to a `grid`).
